code/modules/mob/living/carbon/human/life.dm

```diff
@@ -29,12 +29,10 @@
 			species.handle_unique_behavior(src)
 
 		else //Dead
-			if(!undefibbable && timeofdeath && life_tick > 5)
-				dead_ticks ++
-				if(timeofdeath < 5 || check_tod(src) || !is_revivable())	//We are dead beyond revival, or we're junk mobs spawned like the clowns on the clown shuttle
-					set_undefibbable()
-				else
-					med_hud_set_status()
+			dead_ticks ++
+			if(dead_ticks > TIME_BEFORE_DNR)
+				set_undefibbable()
+			med_hud_set_status()
```

Comment:
`med_hud_set_status()` will be called each Life() on not in stasis dead people. Previously only once, the tick `udefibbable` is set.

Reply:
Also dead_ticks will accumulate infinitely.

Reply:
- Also dead_ticks will accumulate infinitely.
No, cause mob stop processing when its set to undefibbable 

-med_hud_set_status() will be called each Life() on not in stasis dead people. Previously only once, the tick
med_hud was already called each life, since it's called in set_undefib

This needs an early return though, and med_hud_set_status could use a better structure

Reply:
Add at least an else here so medhud isnt updated twice due to set_undefibbable